test/test_ssl_service.py

```diff
@@ -0,0 +1,92 @@
+import io
+import unittest
+from contextlib import redirect_stderr
+from types import SimpleNamespace
+
+from conjur.ssl_service import SSLService
+
+GITHUB_FINGERPRINT = "5F:3F:7A:C2:56:9F:50:A4:66:76:47:C6:A1:8C:A0:07:AA:ED:BB:8E"
```

Comment:
Note that the certificate is valid until 10 May 2022, so in 1.5 years we will have to fix this.
Add a comment on this test with this date to make sure why it starts to fail.